pkg/broker/statestorage/statestorage.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	maintenanceProjectName = "Atlas Service Broker Mainentance"
+	maintenanceProjectName = "Atlas Service Broker Maintenance"
```

Comment:
Meeting discussion: shall we _not_ make this part of the change to avoid having to make it backwards-compatible

Reply:
Also it's possible to rename this project by hand - much easier than migrating all data